src/components/radios/RadioContext.ts

```diff
@@ -11,6 +11,7 @@ export type IRadiosContext = {
 };
 
 export const RadiosContext = createContext<IRadiosContext>({
+  /* eslint-disable @typescript-eslint/no-empty-function */
```

Comment:
Looking at the type definition for this component, none of these functions are optional. Do we even need to create empty defaults for them?

Reply:
If they aren't supplied, Typescript will throw an error as they are required and wouldn't be defined